packages/mongodb-memory-server-core/src/__tests__/MongoMemoryServer-test.ts

```diff
@@ -72,7 +72,7 @@ describe('MongoMemoryServer', () => {
       const mongoServer = new MongoMemoryServer({ autoStart: false });
 
       await expect(mongoServer.ensureInstance()).rejects.toThrow(
-        'Database instance is not running. You should start database by calling start() method. BTW it should start automatically if opts.autoStart!=false. Also you may provide opts.debug=true for more info.'
+        'Ensure Instace failed to start an instance!'
```

Comment:
Should `Instace` be `Instance`? Typo?

Reply:
Thanks, fixed in 46799609b061790b88741e47a4d794a6b24cdcad